src/components/IntroContent/IntroContent.styles.js

```diff
@@ -0,0 +1,22 @@
+import { css } from 'lib/utils/media-queries';
+import { Color, Space } from 'lib/theme';
+
+const styles = {
+  root: css({
+    marginRight: [`${Space.S20}px`, '', 'auto'],
+  }),
```

Comment:
question since haven't used the media queries yet - wondering if the middle query inherits the desktop or mobile styles? or was this intentionally left blank until we have a design for this breakpoint?

Reply:
Anything that has no value, inherits the value before it in array. So here the middle inherits the mobile styles. This was intentional because we have no tablet style yet. I think it would probably be fine to keep it as is, unless they make changes for a tablet design. Thanks!